mypy.ini

```diff
@@ -1,5 +1,11 @@
 [mypy]
 strict = True
 
+[mypy-etest]
+allow_untyped_decorators = True
+
+[mypy-etest.build]
+allow_untyped_decorators = True
+
```

Comment:
Shouldnt this be added in #93 instead?

Reply:
Nope, the build command is already in master just as a stub and this is required to ensure all checks pass on this pull request.  This is where this belongs.